code/modules/antagonists/heretic/items/eldritch_painting.dm

```diff
@@ -0,0 +1,448 @@
+	var/sensor_type = /datum/proximity_monitor/advanced/eldritch_painting
+	// Set to false since we don't want this to persist
+	persistence_id = FALSE
+
+// Mood applied for ripping the painting
+/datum/mood_event/eldritch_painting
+	description = "YOU, I SHOULD NOT HAVE DONE THAT!!!"
+	mood_change = -6
+	timeout = 3 MINUTES
+
+/obj/structure/sign/painting/eldritch/Initialize(mapload, dir, building)
+	if(sensor_type)
+		painting_proximity_sensor = new sensor_type(_host = src, range = 7, _ignore_if_not_on_turf = TRUE)
+	return ..()
+
+/obj/structure/sign/painting/eldritch/wirecutter_act(mob/living/user, obj/item/I)
+	if(!user.can_block_magic(MAGIC_RESISTANCE))
+		user.add_mood_event("ripped_eldritch_painting", /datum/mood_event/eldritch_painting)
+		to_chat(user, span_notice("Laughter echoes through your mind...."))
+	QDEL_NULL(painting_proximity_sensor)
+	qdel(src)
+
+// Applies an affect on view
+/datum/proximity_monitor/advanced/eldritch_painting
+	var/applied_trauma = /datum/brain_trauma/severe/pacifism
+	var/text_to_display = "I should not be seeing this..."
+
+/datum/proximity_monitor/advanced/eldritch_painting/field_turf_crossed(atom/movable/crossed, turf/location)
+	if (!isliving(crossed) || !can_see(crossed, host, current_range))
+		return
+	on_seen(crossed)
+
+/datum/proximity_monitor/advanced/eldritch_painting/proc/on_seen(mob/living/carbon/human/viewer)
+	if (!viewer.mind || !viewer.mob_mood || (viewer.stat != CONSCIOUS) || viewer.is_blind())
+		return
+	if (viewer.has_trauma_type(applied_trauma))
+		return
+	if(IS_HERETIC(viewer))
+		return
+	if(viewer.can_block_magic(MAGIC_RESISTANCE))
+		return
+	to_chat(viewer, span_notice(text_to_display))
+	viewer.gain_trauma(applied_trauma, TRAUMA_RESILIENCE_SURGERY)
+	viewer.emote("scream")
+	to_chat(viewer, span_warning("As you gaze upon the painting your mind rends to its truth!"))
+
+
+
+
+// The sister and He Who Wept eldritch painting
+/obj/item/wallframe/painting/eldritch/weeping
+	name = "The sister and He Who Wept"
+	desc = "A beautiful artwork depicting a fair lady and HIM, HE WEEPS, I WILL SEE HIM AGAIN."
+	icon_state = "eldritch_painting_weeping"
+	result_path = /obj/structure/sign/painting/eldritch/weeping
+
+/obj/structure/sign/painting/eldritch/weeping
+	name = "The sister and He Who Wept"
+	desc = "A beautiful artwork depicting a fair lady and HIM, HE WEEPS, I WILL SEE HIM AGAIN. Destroyable with wirecutters."
+	icon_state = "eldritch_painting_weeping"
+	sensor_type = /datum/proximity_monitor/advanced/eldritch_painting/weeping
+
+// These moods are here to check hallucinations and provide easier user feedback
+
+/datum/mood_event/eldritch_painting/weeping
+	description = "HE IS HERE, AND HE WEEPS!"
+	mood_change = -3
+	timeout = 10 SECONDS
+
+/datum/mood_event/eldritch_painting/weeping_heretic
+	description = "Oh such arts! They truly inspire me!"
+	mood_change = 5
+	timeout = 5 MINUTES
+
+/datum/mood_event/eldritch_painting/weeping_withdrawl
```

Comment:
don't mood events go in their own file?

Reply:
I checked and from what I can tell the traitor demoraliser posters had the mood event by the item/proximity sensor